src/main/java/fruitshop/db/Storage.java

```diff
@@ -0,0 +1,20 @@
+package fruitshop.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> storage = new HashMap<>();
+
+    public void put(String fruit, Integer quantity) {
+        storage.put(fruit, quantity);
+    }
+
+    public Integer getQuantity(String fruit) {
+        return storage.getOrDefault(fruit, 0);
+    }
+
+    public Map<String, Integer> getStorage() {
+        return storage;
+    }
```

Comment:
According to the encapsulation principle, the `storage` map should be private. Moreover, direct access to the map allows for modification which can lead to an inconsistent state. Consider providing methods to interact with the storage data without exposing the map directly.